daemon/links/links.go

```diff
@@ -22,16 +22,17 @@ type Link struct {
 	Ports []nat.Port
 }
 
+// EnvVars generates environment variables for the linked container
+// for the Link with the given options.
+func EnvVars(parentIP, childIP, name string, env []string, exposedPorts map[nat.Port]struct{}) []string {
+	return NewLink(parentIP, childIP, name, env, exposedPorts).ToEnv()
+}
+
 // NewLink initializes a new Link struct with the provided options.
 func NewLink(parentIP, childIP, name string, env []string, exposedPorts map[nat.Port]struct{}) *Link {
```

Comment:
I might open a PR to deprecate and/or move these internal (`NewLink` and `Link`) because nothing is using these, and they're only there to assist the "EnvVars" function now